resources/views/docs/mobile/1/the-basics/system.md

```diff
@@ -81,29 +83,6 @@ if ($token = System::getPushNotificationsToken()) {
 Once you have the token, you may use it from your server-based applications to trigger Push Notifications directly to
```

Comment:
This raised a couple of questions:

- Should these tokens be stored on the server?
- Could a user (on the server) have multiple tokens associated with it if they use multiple devices?
- How to handle tokens when a user logs out and logs back in on with a different account on the device?

Can you give a brief explainer on what's involved to handle messages server side?